app/controllers/work_package_types/subject_configuration_tab_controller.rb

```diff
@@ -43,7 +43,7 @@ def update
                             .call(patterns: build_patterns(permitted))
 
       if result.success?
-        redirect_to edit_type_subject_configuration_path(@type), status: :see_other
+        redirect_to edit_type_subject_configuration_path(@type), notice: I18n.t(:notice_successful_update)
```

Comment:
While I usually hate these, but should we distinguish between request formats here? Namely `html` and `turbo_stream` (not sure if it's called exactly that).

What's being removed is `status: :see_other`, which was surely introduced because at some point the controller responded to a turbo request. Now we set a flash alert instead, which would pretty surely behave badly if the request was a partial update through turbo streams, because it would mean that the success banner appears later in the flow, when the user doesn't need to see it anymore (to be confirmed).

Long story short: Depending on whether this controller is called through turbo streams or as a regular HTML request, we seem to have different expectations on how it should behave.

Reply:
Let me at least assure you of two things:

- `status: :see_other` was pretty sure added here due to a copy&paste
- all the other form controllers of the work package type edit routes use this line without any status
- IIRC `:see_other` is mostly needed, if we do a turbo_frame request. On turbo streams, we use our helpers, called something like `update_with_turbo_streams`, so IMHO we do not need to handle different request formats here. At least not for now, as this controller does not do any turbo stream updates.

Reply:
I might have mixed up turbo frame vs stream in my text above. Anyways, fine for me, it just looked suspicious (as if we jumped from one implementation to another).